### PR TITLE
使用prop-types定义属性,避免在0.49.0上崩溃

### PR DESCRIPTION
**Pull request by chenmo230**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
     "react-native": "=0.26.0",
     "react-native-mock": "=0.2.5",
     "sinon": "=1.17.4",
-    "sinon-chai": "=2.8.0"
+    "sinon-chai": "=2.8.0",
+    "prop-types": "^15.6.0"
```
Comment (RealOrangeOne) [inline package.json:41]: This should probably be moved to `peerDependencies`, as it's required at runtime for the application

**Review by RealOrangeOne — CHANGES_REQUESTED** [review]

**Comment by joshhornby** [comment]
Can this be merged?

**Comment by RealOrangeOne** [comment]
@joshhornby Unfortunately not, until my requested changes are made anyway

**Comment by joshhornby** [comment]
@RealOrangeOne I've made the changes on a new PR https://github.com/RealOrangeOne/react-native-busy-indicator/pull/22

**Comment by joshhornby** [comment]
Actually it looks like this PR is doing it correctly:

https://github.com/facebook/prop-types#how-to-depend-on-this-package
https://github.com/facebook/prop-types/issues/44

So I believe you can close my PR and merge this one.

**Comment by joshhornby** [comment]
I've tested and this is working correctly.

**Comment by RealOrangeOne** [comment]
Ohh, I didn't know that, good to know! Just need the tests to pass now (I assume it's just the linter)

**Comment by joshhornby** [comment]
@RealOrangeOne All tests are passing for this PR 👍

**Comment by RealOrangeOne** [comment]
Oh, I read the wrong `X`, damn you GitHub!

**Review by RealOrangeOne — APPROVED** [review]

**Comment by joshhornby** [comment]
Thanks for merging, can you cut a new release as well? @RealOrangeOne

**Pull request merged**